packages/@coorpacademy-components/src/variables/colors.ts

```diff
@@ -1,13 +1,9 @@
-type Colors = {
-  gray: string;
-  negative: string;
-  positive: string;
-  cm_grey_800: string;
-};
-
-export const COLORS: Colors = {
+export const COLORS = {
   gray: '#EAEAEB',
   negative: '#ed3436',
   positive: '#35CC7F',
-  cm_grey_800: '#171721'
-};
+  cm_grey_800: '#171721',
+  cm_primary_blue: '#0061FF'
```

Comment:
idealy, you can replace all `'#0061FF'` hardcoded accross the code by an import
(this is bonus)